pypicloud/util.py

```diff
@@ -80,7 +80,7 @@ def normalize_metadata_value(value: Union[str, bytes]) -> str:
         value = value.decode("utf-8")
     if isinstance(value, str):
         value = "".join(c for c in unicodedata.normalize("NFKD", value) if ord(c) < 128)
-    return value
+    return re.sub(r"\s+", " ", value)
```

Comment:
This will replace all whitespace characters with `" "`. Could we instead do 
```python
re.sub(r"  +", " ", value)
```
to only replace instances of two-or-more spaces with a single space?

Reply:
I tried the presence of other whitespace characters, the same problem would occur if \t existed, and the presence of \n and \r would succeed, but the characters after the whitespace were removed. 
And observing that only X-AMZ-meta-summary affects the request, the values processed include the following.
```
'x-amz-meta-hash-sha256': '71eccb33ac8b2584c86f36f8ebfbe72c0a98022b576f411ab05773343e4e2cac', 
 'x-amz-meta-hash-md5': 'f562366338d015df143c7afacabdee44', 
 'x-amz-meta-summary': ' Python3\tweb , WSGI, Web ', 
 'x-amz-meta-name': 'frontgate', 'x-amz-meta-version': '0.6.39'
```
So I think it's going to be implemented the same way as before.

Reply:
It's the COS of Tencent Cloud.

Reply:
Okay, last request:
This is going to replace all kinds of whitespace with spaces. It would be better if we could preserve the original whitespace if it's not causing issues. I believe this should do it
```python
re.sub(r'(\s)\s+', r'\1', value)
```

Reply:
Okay, that looks better

Reply:
Wow COS really doesn't like whitespace, huh? I guess this is fine. The worst that we expect is some mangling of the summary formatting.